doc/lua_api.txt

```diff
@@ -2291,6 +2304,12 @@ The following functions provide escape sequences:
 
 Spatial Vectors
 ===============
+A spatial vector similar to a position, but instead using
+absolute world coordinates, it uses *relative* coordinates, relative to
+no particular point.
+
+Internally, it is implemented as a table with the 3 fields
+`x`, `y` and `z`. Example: `{x = 0, y = 1, z = 0}`.
```

Comment:
In this case I'd also adjust `vector.new` accordingly which does not need the entire explanation any more.